py3status/py3.py

```diff
@@ -40,6 +51,41 @@ def __repr__(self):
         return 'None'
 
 
+class HTTPResponse:
+    """
+    This class is like a simplified version of `urllib3.response.HTTPResponse`.
+
+    The HTTPResponse object provides status, data, and json attributes.
+
+    @TODO: Implement the header attribute
+    """
+
+    def __init__(self, resp_obj):
+        self._resp_obj = resp_obj
+        self._data = None
+        self._json = None
+
+    def __getattr__(self, name):
+        # If trying to access an attribute that doesn't exist in this class,
+        # try accessing that attribute from the response object.
+        return getattr(self._resp_obj, name)
```

Comment:
Makes sense.  I added it initially so that all of the `I/O`-type methods would work (`read()`, `readline()`), but we don't need those exposed with the functions we have defined.